### DISCUSSION TITLE
Install for Filament v3

### DISCUSSION THREAD

**Discussion by HomaEEE** [Z3d0X/filament-fabricator#85] [Q&A] [answered]
How install to v3 of Filament - after farbicator:install don't publish resource and model

**Comment by Z3d0X** [comment, marked as answer]
I think I forgot to mention this in the docs

Register a `FilamentFabricatorPlugin` instance in your Panel provider:

```php
public function panel(Panel $panel): Panel
{
    return $panel
        // ...
        ->plugins([
            FilamentFabricatorPlugin::make(),
        ]);
}
```

**Reply by HomaEEE** [reply]
sorry, completely forgot about the panel

**Comment by aasuncion-fol** [comment]
I did add the plugin in the panel but encountered this error. Please help 

Class "App\Providers\Filament\FilamentFabricatorPlugin" not found

**Reply by Z3d0X** [reply]
Please import the plugin class on the top of your service provider...
```php
use Z3d0X\FilamentFabricator\FilamentFabricatorPlugin;
```
